Packs/DeCYFIR/Integrations/decyfir/decyfir.py

```diff
@@ -0,0 +1,483 @@
+
+VAR_ATTACK_SURFACE = "attack-surface"
+VAR_IMPERSONATION_AND_INFRINGEMENT = "impersonation-and-infringement"
+VAR_DATA_BREACH_AND_WEB_MONITORING = "data-breach-and-web-monitoring"
+VAR_SOCIAL_AND_PUBLIC_EXPOSURE = "social-and-public-exposure"
+
+# ATTACK SURFACE
+VAR_OPEN_PORTS = "open-ports"
+VAR_IP_VULNERABILITY = "ip-vulnerability"
+VAR_CONFIGURATION = "configuration"
+VAR_CLOUD_WEAKNESS = "cloud-weakness"
+VAR_IP_REPUTATION = "ip-reputation"
+VAR_CERTIFICATES = "certificates"
+
+VAR_ATTACK_SURFACES_SUB_TYPES: list = [VAR_OPEN_PORTS, VAR_IP_VULNERABILITY, VAR_CONFIGURATION, VAR_CLOUD_WEAKNESS,
+                                       VAR_IP_REPUTATION, VAR_CERTIFICATES]
+
+# IMPERSONATION & INFRINGEMENT
+VAR_DOMAIN_IT_ASSET = "domain-it-asset"
+VAR_EXECUTIVE_PEOPLE = "executive-people"
+VAR_PRODUCT_SOLUTION = "product-solution"
+VAR_SOCIAL_HANDLERS = "social-handlers"
+
+VAR_IMPERSONATION_AND_INFRINGEMENT_SUB_TYPE: list = [VAR_DOMAIN_IT_ASSET, VAR_EXECUTIVE_PEOPLE, VAR_PRODUCT_SOLUTION,
+                                                     VAR_SOCIAL_HANDLERS]
+
+# DATA BREACH AND WEB MONITORING
+VAR_PHISHING = "phishing"
+VAR_RANSOMWARE = "ransomware"
+VAR_DARK_WEB = "dark-web"
+
+VAR_DATA_BREACH_AND_WEB_MONITORING_SUB_TYPES: list = [VAR_PHISHING, VAR_RANSOMWARE, VAR_DARK_WEB]
+
+# SOCIAL AND PUBLIC EXPOSURE
+VAR_SOURCE_CODE = "source-code"
+VAR_MALICIOUS_MOBILE_APPS = "malicious-mobile-apps"
+VAR_CONFIDENTIAL_FILES = "confidential-files"
+VAR_DUMPS_PII_CII = "dumps-pii-cii"
+VAR_SOCIAL_THREAT = "social-threat"
+
+VAR_SOCIAL_AND_PUBLIC_EXPOSURE_SUB_TYPES: list = [VAR_SOURCE_CODE, VAR_MALICIOUS_MOBILE_APPS, VAR_CONFIDENTIAL_FILES,
+                                                  VAR_DUMPS_PII_CII, VAR_SOCIAL_THREAT]
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def say_hello(self, name):
+        return f'Hello {name}'
+
+    def get_severity(self, risk_score: int):
+        if risk_score > 8:
+            return IncidentSeverity.CRITICAL
+        elif risk_score > 5:
+            return IncidentSeverity.HIGH
+        elif risk_score >= 3:
+            return IncidentSeverity.MEDIUM
+        elif risk_score >= 1:
+            return IncidentSeverity.LOW
+        else:
+            return IncidentSeverity.UNKNOWN
+
+    def request_decyfir_api(self, endpoint) -> str:
+        response = requests.get(f"{endpoint}")
+        if response.status_code == 200:
+            if len(response.text) > 0:
+                return json.dumps(response.json())
+        else:
+            print(f"Error: =>  {response.status_code} error with API request")
+            return ""
+
+    def get_decyfir_data(self, after_val: int, is_first_fetch: bool, org_api_key: str, incident_type: str) -> str:
+        endpoint_env = f"{QA_API_PATH}"
+
+        size: int = MAX_INCIDENTS_TO_FETCH
+        if is_first_fetch:
```

Comment:
Why does the size changes between fetch calls?

Reply:
Yes, the first time we need the maximum size  of data can get, after the first fetch getting the default size.